lib/page.rb

```diff
@@ -22,7 +22,8 @@ def self.has_css?(css, options={})
       wait = Selenium::WebDriver::Wait.new(:timeout => 5)
       begin
         wait.until {Driver.driver.find_element(:css, css).enabled?}
-      rescue Exception
+      rescue Exception => exception
+        Log.debug("page.has_css? is false because this exception was rescued: #{exception}")
```

Comment:
Can we prefix these log messages with "[GRIDIUM::Page] " I like being able to see output like that in debug mode.